feature/experimental/system/gnmi/benchmarking/ate_tests/full_configuration_replace_test/README.md

```diff
@@ -0,0 +1,27 @@
+# gNMI-1.2: Benchmarking: Full Configuration Replace 
+
+## Summary
+
+Measure performance of full configuration replace.
+
+## Procedure
+
+Configure DUT with:
+ - Maximum number of interfaces to be supported.
+ - Maximum number of BGP peers to be supported.
+ - Maximum number of IS-IS adjacencies to be supported.
+Measure time required for Set operation to complete. 
+Modify descriptions of a subset of interfaces within the system.
+Measure time for Set to complete.
+
+Notes:
+This test does not cover entirely converged system, simply replacing
```

Comment:
This sentence is a little hard to parse. Maybe something like "This test does not measure an the time to an entirely converged state, only to completion of the gNMI update." ?

Reply:
Re-worded it per suggestion. Thanks.